meroxa/resource_connector_test.go

```diff
@@ -77,6 +77,67 @@ func TestAccMeroxaConnector_WithoutPipeline(t *testing.T) {
 	})
 }
 
+func TestAccMeroxaConnector_WithInvalidName(t *testing.T) {
```

Comment:
This function is pretty much copied over [from here](https://github.com/meroxa/platform-api/blob/947d72432b519b43c772dd6d4ea5196333373267/mediator/connector/shared.go#L18-L43). I'd like to consolidate this to a shared library of sorts. Thoughts?